### ISSUE TITLE
没有已经训练好的模型吗？

### ISSUE THREAD

**Issue by aawwwwwhahah** [4DVLab/DexGrasp-Anything#22]
需要自己下载数据集重新训练吗？

**Comment by ymzhong66** [comment]
有训练好的ckpt呀  你可以去下载的

**Comment by aawwwwwhahah** [comment]
能给一下下载链接吗，我在库里没有找到，谢谢你

**Comment by ymzhong66** [comment]
你确定吗？  你是第一次用github 吗？

<img width="1049" height="555" alt="Image" src="https://github.com/user-attachments/assets/a185935e-2e2a-47ce-b4cd-7254504d516f" />这里不是有这么多吗？

**Comment by aawwwwwhahah** [comment]
其实我是想要你训练的checkpoints，但我并没有找到

**Comment by ymzhong66** [comment]
啥意思？   你是说所有数据集的吗？ 那个目前我也没有了

**Comment by ymzhong66** [comment]
你有什么不明白的可以给我发邮件  我来帮助你解决？

**Comment by aawwwwwhahah** [comment]
谢谢你了，不过我没什么问题了。

**Comment by aawwwwwhahah** [comment]
我在训练过程中确实遇到了问题，我设置的batchsize是128，使用的是单张3090进行训练，数据集使用的是Dexgraspanyting，模型是unet_grasp。
name: grasp_gen_ur
lr: 1e-4
eval_interval: 1
eval_visualize: 1

train:
  batch_size: 128
  num_workers: 0
  num_epochs: 3000 #3000 for small dataset, 100 for large dataset(1M+)
  log_step: 100
运行结果在下面
concatenating texture: may result in visual artifacts
Epoch 1/3000:   9%|███▋                                       | 2303/26575 [33:31<5:53:43,  1.14it/s, loss=0.799]
训练速度是否太慢了？一次就需要训练6个小时，请问有没有加快训练的方法，你当初是训练了多久呢？

**Comment by ymzhong66** [comment]
这个 数据集确实是很庞大的3.4M的数据  单卡可能训练起来就是会很慢 这个我用4张A卡训了1周

**Comment by aawwwwwhahah** [comment]
能提供你训练好的权重文件吗，我们实验室的算力不太够。

**Comment by ymzhong66** [comment]
我原来的那个在整理到这个git之后  集群占用太多清理了   你非常急需吗？假如不是特别急需  可以等我  cvpr之后 我重训一版本给你？

**Comment by aawwwwwhahah** [comment]
目前不是很急，非常感谢你的慷慨相助




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Yiming ***@***.***&gt;; 
发送时间: 2025年10月14日(星期二) 晚上8:08
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [4DVLab/DexGrasp-Anything] 没有已经训练好的模型吗？ (Issue #22)












ymzhong66 left a comment (4DVLab/DexGrasp-Anything#22)
 
我原来的那个在整理到这个git之后  集群占用太多清理了   你非常急需吗？假如不是特别急需  可以等我  cvpr之后 我重训一版本给你？

**Comment by ymzhong66** [comment]
右面有时间我重新训练一版本全部的数据混合的吧 你可以提醒我一下
